### ISSUE TITLE
Not Available in French App Store

### ISSUE THREAD

**Issue by hugo-abreu** [MobiusSync/MobiusSync#27]
I cannot find Mobius Sync through the app store, and when I open the link through the QR code - I get the following error:

App Not Available: This app is currently not available in your country or region.

I am in France: is it an oversight or is there a reason it is not available in France?

Thanks

**Comment by sjpickup** [comment]
The app is excluded from the French app store because there are particular restrictions on encryption that requires quite a complex and time-consuming approval process that I cannot justify at this stage, sorry!

See also https://github.com/MobiusSync/MobiusSync/discussions/16

**Comment by alexandresoro** [comment]
Hello @sjpickup ,

Is there anything we could do to help with this process?

I was not aware such paperwork still existed. 
Commercial apps like Resilio or Mega and even open source apps like Cryptomator or CryptCloudViewer - which is an app developed by a single person in Japan, who I assume does not speak French - are available on the French store without restriction. So I am honestly surprised that apps like the latter did fill this paperwork, as I do not see any fundamental difference in terms of cryptography with Mobius.

Appreciate your work, and looking forward to hopefully use it in France as well!

**Comment by hugo-abreu** [comment]
Anything new? @sjpickup

**Comment by sjpickup** [comment]
I can't speak for other apps, I can only refer you to:

- https://www.ssi.gouv.fr/en/regulation/cryptology/how-to-submit-an-application/

- https://cryptomator.org/blog/2016/06/16/indepth-french-app-store/

If MöbiusSync was a native Swift app it could use the built-in iOS APIs for encryption, but since the Syncthing core is in Golang it effectively implements its own encryption.

Logistically it's doable, it's only a question of priorities.

**Comment by alexandresoro** [comment]
Thanks for the update!

I had a quick look at the forms and the documents, and indeed it looks like extensive administrative paperwork.
I understand why you don't see this as a priority right now.
Should you need any assistance on this one day, me or someone fluent in French over here would probably be glad to help!

**Comment by fjolliton** [comment]
The BlinkShell app (https://github.com/blinksh/blink) had the same issue not so long ago. Maybe they can provide you with some insights for the legal process.

**Comment by alxgda** [comment]
Also available for help if needed.

**Comment by Elaws** [comment]
Hello,

I am very interested by this app : is there any way to get the IPA file so we can install it on iPad, and possibly pay you to get the unlimited storage access ?

Thank you very much

**Comment by Elaws** [comment]
Hello,

Any news on this please ?

Thanks !

**Comment by Elaws** [comment]
Hello,

I'm really sorry to insist, but is it possible to get some news on that issue ? 

Thanks a lot

**Comment by Ticonderoga** [comment]
I’m also very interested by the availability of mobius sync in the French market.

tia

**Comment by JPenuchot** [comment]
Hello, I'm very interested too, and willing to help for the paperwork as well :)

**Comment by JPenuchot** [comment]
I sent an e-mail to the ANSSI asking them for a translation of the document. They gave me a courtesy translation of the document, so you can use it as a reference to fill out the **French** version. You **can** fill it up in English though.

Here are the docs as they sent them to me: [Attachments-RE Formulaire Anglais pour la demande d'autorisation relative aux services de chiffrement.zip](https://github.com/MobiusSync/MobiusSync/files/10580778/Attachments-RE.Formulaire.Anglais.pour.la.demande.d.autorisation.relative.aux.services.de.chiffrement.zip)

tl;dr: fill the **French/untranslated** document in **English**, using the translated document as a reference only. It should be accepted as long as the form is correct.

**Comment by JPenuchot** [comment]
@sjpickup I hope this can help you, the form itself shouldn't take too long to redact and I'm pretty sure they will give you a positive response quite quickly.

**Comment by lauhub** [comment]
@sjpickup If you need any help in completing these forms, just let me know, I will be glad to help too

**Comment by leafblowersama** [comment]
Hey,

Chiming in here, I'm super disappointed to see that this appears to be unavailable strictly for France and only France, and I'd really like to be able to use this app (And Apple being Apple, changing region in the app store is ... ugh).

I'm also available to help in whatever way I can to fill out the forms if you like, it'd just be a shame to leave this hanging.
Let me know

**Comment by Salamafet** [comment]
Soon, the Digital Markets Act will come into effect in Europe.
Apple must open third-party application installation.

We need to wait a little bit more and sooner I hope we'll be able to install Möbius Sync without going through the app store. 🤞

**Comment by JPenuchot** [comment]
It would be nice to have it from the store to keep the app up-to-date though.

The ANSSI form isn't that long, it's not too technical, and we even provided an English version of the form but Mobius project members don't seem to care about it.

**Comment by jalberto** [comment]
It is not available in the Spanish one either, so I guess is an EU level thingy

**Comment by JPenuchot** [comment]
No, as we've discussed French law mandates companies to redact a form from the French agency for the security of information systems (ANSSI) before they put a paid app with encryption features on the market. This is what's blocking Mobius from selling their app on the iOS store in France despite the form being pretty basic.

**Comment by goguelnikov** [comment]
Hello Guys,
Any new on this subject?
I am also interested and ready to pay for the app if I can download it ;-)
I could potentially download it from the US App Store, but it's a mess...
Can you / will you do something ?
Best

**Comment by Negryl** [comment]
I’m also paying. Are there plans to get this French approval?

Other apps such as Cryptomator and Resilio are available in France.

**Comment by Elaws** [comment]
I heard about third party stores with iOS & iPadOS 18 : does this mean there is hope for MobiusSync in France ?

Thank you very much.

**Comment by Acelya-9028** [comment]
What is this bullshit lol? Why do you care about French approval U_U? Fucking bullshit T_T

**Comment by safaiyeh** [comment]
I ran into this with my own app and built a tool to generate the required docs automatically: https://www.novastore.dev

Hope it saves someone else the paperwork headache.

Let's get your app in France let me know if you got questions

**Comment by RSLLES** [comment]
Hello,

I would be happy to purchase your app if it were available in France. I would greatly appreciate it if you could complete the necessary administrative steps to make it accessible here (I haven't found any alternative that matches the quality of your product)

Thank you !

@safaiyeh your app looks down

**Comment by Elaws** [comment]
I heard about another iOS syncthing client called Syntrain, but apparently same problem as MöbiusSync (not available in Europe).

**Comment by Elaws** [comment]
I heard about another iOS syncthing client called Synctrain, apparently available in Europe except France since end of last year.

On their website, developer say they are eager to work with people knowing how to fill the specific form for specific country so maybe @safaiyeh you could get in touch with them : 
https://t-shaped.nl/synctrain-support 

After nearly half a decade of waiting, maybe French user have a slight hope to see a Syncthing client on their Apple devices !

**Comment by ianvg** [comment]
I'd also like to voice my input and say that access to this app in France/EU would be a game changer!

**Comment by Baggio94** [comment]
Any news ?

**Comment by alexandresoro** [comment]
> I heard about another iOS syncthing client called Synctrain, apparently available in Europe except France since end of last year.
> 
> On their website, developer say they are eager to work with people knowing how to fill the specific form for specific country so maybe [@safaiyeh](https://github.com/safaiyeh) you could get in touch with them : https://t-shaped.nl/synctrain-support
> 
> After nearly half a decade of waiting, maybe French user have a slight hope to see a Syncthing client on their Apple devices !

FYI, it looks like Synctrain is going to be soon available in France: https://github.com/pixelspark/sushitrain/discussions/128#discussioncomment-16526453
They got approved by Apple following their process with ANSSI. Cherry on the cake, their app is open source.
